src/Dialogs/AdvancedTips.less

```diff
@@ -0,0 +1,44 @@
+@import (less) "../base.less";
+
+.advanced_tips {
+  box-sizing: border-box;
+  background-color: @gray-20;
+
+  //override parent padding
+  margin-left: -15px;
+  margin-right: -15px;
+  margin-bottom: -15px;
+  margin-top: 15px;
+
+  padding: 15px;
+
+  .tips {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+  .tip {
+    flex-basis: 31%;
+    padding-right: 2%;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    .example-input {
+      box-sizing: border-box;
+      width: 48%;
+      color: @gray-80;
+      display:inline-block;
+      &.first {
```

Comment:
The other way to do this is [`input:first-of-type`](https://www.w3schools.com/cssref/sel_first-of-type.asp)